### ISSUE TITLE
[Feature Request] L2 cache options with SizeLimit

### ISSUE THREAD

**Issue by xubinzheng** [AzureAD/microsoft-identity-web#1894]
**Is your feature request related to a problem? Please describe.**
Today the OBO token cache is eating our Redis memory.

**Describe the solution you'd like**
We ware wondering can MSAL has an option to put a size limit on the L2 distributed token cache?
Just like the L1CacheOptions.SizeLimit: https://github.com/AzureAD/microsoft-identity-web/blob/704cd3f8a8db8833888a55623c7d8e686ef6b424/src/Microsoft.Identity.Web.TokenCache/Distributed/MsalDistributedTokenCacheAdapterOptions.cs#L22

**Comment by jmprieur** [comment]
@xubinzheng : this is done by the cache eviction policies., for instance this property: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.distributed.distributedcacheentryoptions.slidingexpiration?view=dotnet-plat-ext-6.0#microsoft-extensions-caching-distributed-distributedcacheentryoptions-slidingexpiration

**Comment by xubinzheng** [comment]
@jmprieur yes, but it doesn't have an option to set the SizeLimit. Who should own this feature ask?
